src/main/java/com/iota/iri/service/dto/GetNeighborsResponse.java

```diff
@@ -42,8 +42,10 @@ public String getConnectionType() {
 
         public static Neighbor createFrom(com.iota.iri.network.Neighbor n) {
             Neighbor ne = new Neighbor();
+            java.net.InetSocketAddress address = n.getAddress();
```

Comment:
there is a `Neighbor` field also named `address` which is a string in the same file above - better to change the name to avoid confusion

Reply:
@cwarner818 

    public static Neighbor createFrom(com.iota.iri.network.Neighbor n) {
        Neighbor ne = new Neighbor();
        java.net.InetSocketAddress socketAddress = n.getAddress();
        int port = n.getPort();
        ne.address = socketAddress.getHostString() + ":" + port;
        ne.ip = socketAddress.getAddress().getHostAddress();
        ...
    }